pkg/cmd/init.go

```diff
@@ -170,7 +208,7 @@ func NewHelmInstall(version string, ioStreams cmdutil.IOStreams) (*action.Instal
 		kube.GetConfig(cmdutil.GetKubeConfig(), "", types.DefaultOAMNS),
 		types.DefaultOAMNS,
 		os.Getenv("HELM_DRIVER"),
-		ioStreams.Infof,
+		info,
```

Comment:
why need this change?

Reply:
this is a log debug switch, so we need to input a debug log function.